### PR TITLE
added some testcases for inheritance with transactionAttribute

### PR DESCRIPTION
**Pull request by LightCZ**
draft for fix when inheriting and using transaction attribute in parrent.
so far everything works as expected + also addes some tests for that

### CODE REVIEW

File: src/Castle.Facilities.AutoTx/TransactionClassMetaInfoStore.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ public TransactionClassMetaInfoStore()
 		internal static Maybe<TransactionalClassMetaInfo> GetMetaFromTypeInner(Type implementation)
 		{
 			Contract.Ensures(Contract.Result<Maybe<TransactionalClassMetaInfo>>() != null);
-
+            var a = implementation.Name == "InheritedMyService";
```
Comment (haf) [inline src/Castle.Facilities.AutoTx/TransactionClassMetaInfoStore.cs]: whitespace

File: src/Castle.Facilities.AutoTx/TransactionInterceptor.cs [hunk 2]
```diff
@@ -81,9 +81,7 @@ void IInterceptor.Intercept(IInvocation invocation)
 			var txManagerC = _Kernel.Resolve<TransactionManager>();
 			ITransactionManager txManager = txManagerC;
 
-			var mTxMethod = _MetaInfo.Do(x => x.AsTransactional(invocation.Method.DeclaringType.IsInterface
-			                                                    	? invocation.MethodInvocationTarget
-			                                                    	: invocation.Method));
+            var mTxMethod = _MetaInfo.Do(x => x.AsTransactional(invocation.MethodInvocationTarget ?? invocation.Method));
```
Comment (haf) [inline src/Castle.Facilities.AutoTx/TransactionInterceptor.cs]: whitespace

**Comment by haf** [comment]
Ok, nice! =)

**Comment by LightCZ** [comment]
Oh sorry that change in TransactionClassMetaInfoStore.cs is done by mistake and i forgot to delete that line.

IN transaction Interceptor i am unsure what do you mean by whitespace. You miss more whitespace?

What do you mean by whitespace . 


BTW: i ll probably link this issue to this: http://stackoverflow.com/questions/29191721/castle-transaction-facility-inheritance-issue

**Comment by haf** [comment]
If you have a look at the diff, you'll see you've used tabs instead of spaces, or spaces instead of tabs (depending on what this codebase was written in).

screen shot 2015-03-23 at 14 27 01

In github it's easy to spot as tabs are formatted as eight spaces. I think you're using spaces when it should be tabs.

You can enable 'View Whitespace' under `Edit > Advanced > View Whitespace` if you want to view it in VS.

**Comment by haf** [comment]
So do you push another commit where you've deleted what you forgot?

**Comment by LightCZ** [comment]
Sure give me a minute. i ll commit fix

**Comment by haf** [comment]
Sweet! ;)

**Comment by LightCZ** [comment]
It should be up and running - https://github.com/LightCZ/Castle.Transactions/commit/8e0f50d7e7392610f173fe4b842b1638ee5008a4

I hope its ok since i use resharper for formatting.

**Comment by haf** [comment]
That looks good!

**Comment by haf** [comment]
Thank you @LightCZ  and welcome to contribute more in the future!

**Pull request merged**